README.md

```diff
@@ -21,7 +21,7 @@ When you run npm install within your project's root folder, the plugin will be a
 
 ```sh
```

Comment:
NIT: Could be `json` instead of shell for better readability if this will get bigger